src/appFunctions.js

```diff
@@ -145,25 +124,21 @@ export function getParentCardArrayByIdChildren(id, cards) {
 }
 
 export function filterStatusesByEnabled(statuses) {
-    
-    return statuses.filter(status => (status.enabled));
+    return statuses.filter(({enabled}) => enabled);
 }
 
 export function getStatusNameAdditionOfSettings(name, statuses) {
+    return statuses.find(status => status.name === name) ? name : `${name} (deleted)`;
+}    
 
-    const objectStatus = statuses.find(status => status.name === name);
-
-    if (objectStatus) {
-
-        if (!objectStatus.enabled) {
-
-            return `${name} (disabled)`
-        }
-    }
-    else {
+export function getStateStatus(name, statuses) {
+    return !statuses.find(status => status.name === name).enabled;
```

Comment:
it will break if there is no such status since `.find` returns and object or undefined
undefined.enabled will throw